test/setup.js

```diff
@@ -0,0 +1,13 @@
+import { jsdom } from 'jsdom';
+
+global.document = jsdom('');
+global.window = document.defaultView;
+Object.keys(document.defaultView).forEach(property => {
+  if (typeof global[property] === 'undefined') {
+    global[property] = document.defaultView[property];
+  }
+});
+
+global.navigator = {
+  userAgent: 'node.js'
+};
```

Comment:
Setting up file for Full DOM rendering using enzyme `mount` API.
See also: https://github.com/airbnb/enzyme/blob/5ea91d3de12f1681db5892cff5b10fe6cc7e75e8/docs/guides/jsdom.md